src/com/carolinarollergirls/scoreboard/core/impl/TeamImpl.java

```diff
@@ -165,6 +184,7 @@ public void reset() {
             removeAll(Child.ALTERNATE_NAME);
             removeAll(Child.COLOR);
             removeAll(Child.SKATER);
+            removeAll(Child.TIME_OUT);
```

Comment:
This doesn't seem to be working, a Start New Game isn't clearing all the timeouts